flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/utils/EventUtils.java

```diff
@@ -107,7 +104,7 @@ public static boolean createOrUpdateEvent(
                             .withNamespace(target.getMetadata().getNamespace())
                             .endMetadata()
                             .build();
-            client.v1().events().inNamespace(target.getMetadata().getNamespace()).create(event);
+            client.resource(event).createOrReplace();
```

Comment:
What namespace this event will be created into?

Reply:
Oh I see,  whatever you specified on the resource. It's just removes the redundant`.inNamespace()` part.

Reply:
The one in the event metadata